php/class-plugin.php

```diff
@@ -43,6 +43,8 @@ public function init() {
 
 		// Priority is set to 100 so that plugins can attach validation-sanitization filters at default priority of 10.
 		add_action( 'customize_validate_settings', array( $this, 'validate_settings' ), 100 );
+
+		add_action( 'customize_controls_print_footer_scripts', array( $this, 'print_templates' ) );
```

Comment:
This should have a priority of 1 so that we can be sure the template is available.